docs/io.cozy.accounts.md

```diff
@@ -169,27 +169,22 @@ What we aim for:
 }
 ```
 
-### Caisse d'Épargne (Linxo connector)
+### Caisse d'Épargne (Budget Insights connector)
 
-The connectors based on Linxo API are storing specific informations into `data` attribute.
+The connectors based on Budget Insight API are storing specific informations into `data` attribute.
 
 ```json
 {
-  "account_type": "linxo",
+  "account_type": "caissedepargne1",
   "auth": {
-    "folderPath": null,
-    "frequency": "week",
-    "identifier": "0000000000",
-    "secret": "*********"
+    "bankIds": ["1"]
```

Comment:
Not in the nominal case when BI webview will be in production. But yes, it will still be there for old account and will be managed by the banking connectors